docs/install-upgrade/config.md

```diff
@@ -208,10 +90,89 @@ spec:
+There are two users on the switches, `admin` and `operator`. The `operator` user has
+read-only access to `sonic-cli` command on the switches. The `admin` user has
+broad administrative power on the switch. 
+In order to avoid conflicts, do not use the following usernames: `operator`,`hhagent`,`netops`.
+
+### NTP and DHCP
+The control node uses public NTP servers from Cloudflare and Google by default.
+The control node runs a DHCP server on the management network. See the [example
+file](#complete-example-file).
+
+### Control Node
+The control node is the host that manages all the switches, runs k3s, and serves images. 
+The **management** interface is for the control node to manage the fabric 
+switches, *not* end-user management of the control node. For end-user 
+management of the control node specify the **external** interface name.
+
+### Telemetry
+
+There is an option to enable [Grafana
+Alloy](https://grafana.com/docs/alloy/latest/) on all switches to forward metrics and logs to the configured targets using
+[Prometheus Remote-Write
+API](https://prometheus.io/docs/specs/prw/remote_write_spec/) and Loki API. Metrics includes port speeds, counters, 
+errors, operational status, transceivers, fans, power supplies, temperature
+sensors, BGP neighbors, LLDP neighbors, and more. Logs include Hedgehog agent logs.
+
+```{ .yaml title="telemetry.yaml" linenums="1" }
+spec:
+  config:
+    fabric:
+      defaultAlloyConfig:
+        agentScrapeIntervalSeconds: 120
+        unixScrapeIntervalSeconds: 120
+        unixExporterEnabled: true
+        lokiTargets:
+          grafana_cloud: # target name, multiple targets can be configured
+              basicAuth: # optional
+                  password: "<password>"
+                  username: "<username>"
+              labels: # labels to be added to all logs
+                  env: env-1
+              url: https://logs-prod-021.grafana.net/loki/api/v1/push
+        prometheusTargets:
+          grafana_cloud: # target name, multiple targets can be configured
+              basicAuth: # optional
+                  password: "<password>"
+                  username: "<username>"
+              labels: # labels to be added to all metrics
+                  env: env-1
+              sendIntervalSeconds: 120
+              url: https://prometheus-prod-36-prod-us-west-0.grafana.net/api/prom/push
+        unixExporterCollectors: # list of node-exporter collectors to enable, https://grafana.com/docs/alloy/latest/reference/components/prometheus.exporter.unix/#collectors-list
+        - cpu
+        - filesystem
+        - loadavg
+        - meminfo
+        collectSyslogEnabled: true # collect /var/log/syslog on switches and forward to the lokiTargets
+```
+
+Telemetry can be enabled after installation of the fabric. Open the following
+YAML file in an editor on the control node. Modify the fields as needed. Logs
+can be pushed to a grafana instance at the customer environment, or to grafana
```

Comment:
Should move back, above the YAML file. Only the last sentence should stay here.

Reply:
my bad, got a little too enthusiastic